elasticsearch_collectd.py

```diff
@@ -628,10 +643,28 @@ def fetch_stats():
                 parse_index_stats(indexes_json_stats[index_name], index_name)
 
 
+# SSL requests wrapper
+class HTTPSClientAuthHandler(urllib2.HTTPSHandler):
+    def __init__(self, key_file, cert_file):
+        urllib2.HTTPSHandler.__init__(self)
+        self.key_file = key_file
+        self.cert_file = cert_file
+
+    def https_open(self, req):
+        return self.do_open(self.get_connection, req)
```

Comment:
naive: Does this need to be closed at some point?

Reply:
Looking at the base class this inherits from, it has a close method which doesn't do anything. 
```
    def close(self):
        # Only exists for backwards compatibility
        pass
```
So it doesn't seem like it's needed with this library. Good question though, normally I'd expect that to be needed.

Reply:
Cool, thanks for looking into it